packages/landingpage-new/components/shared/navigationMenu.tsx

```diff
@@ -49,34 +58,24 @@ export function NavigationMenu() {
   }
 
   return (
-    <aside className={styles.aside}>
+    <aside className={styles.aside + ' menu'}>
       <nav className={styles.navigationMenu}>
         <h5>REACT GUIDE</h5>
```

Comment:
This should be provided by props as it could also be used by the angular guide in the near future